waterdrop-core/src/main/scala/io/github/interestinglab/waterdrop/action/Jdbc.scala

```diff
@@ -0,0 +1,108 @@
+package io.github.interestinglab.waterdrop.action
+
```

Comment:
Please add the `license` and adjust the `package` , now the `package` has been modified to `org.apache.seatunnel`

Reply:
Sorry, I read the wrong branch. The `package` of the `master branch` has not been modified yet, but please add the `license`, Thanks

Reply:
Thanks for the reminder, license has been added